gapic-generator-java/src/test/proto/echo.proto

```diff
@@ -44,6 +44,7 @@ service Echo {
   // This service is meant to only run locally on the port 7469 (keypad digits
   // for "show").
   option (google.api.default_host) = "localhost:7469";
+  option (google.api.api_version) = "v1_20230601";
```

Comment:
Depending on if `(google.api.api_version)=""` if allowed or not, can you add test case for that?

Ignore this comment if the above is not allowed.

Reply:
Also, can you add a test case where there are multiple services being parsed and each service has a different api_version? It could also be where it doesn't necessarily follow `v{major version}_date` format. Maybe one could have a version like `(google.api.api_version)=5.3` or something